src/ios/CodePush.m

```diff
@@ -235,7 +235,9 @@ - (BOOL)loadPackage:(NSString*)packageLocation {
 }
 
 - (void)loadURL:(NSURL*)url {
-    [((UIWebView*)((CDVViewController *)self.viewController).webView) loadRequest:[NSURLRequest requestWithURL:url]];
+    // Cast to a UIWebView here to enable call to loadRequest.
+    // Both UIWebView and WKWebview share this call, so regardless of which type the returned webview is, this will work.
+    [(UIWebView*)self.webView loadRequest:[NSURLRequest requestWithURL:url]];
```

Comment:
What versions of cordova-ios did you use for testing? We currently support 3.9.0+ so we need to make sure this will compile on all versions. I remember we had a compilation issue with 4.0.1 in the past.